src/components/wallet/card/wallet-card.tsx

```diff
@@ -165,8 +165,15 @@ const WalletCard = ({
         );
     };
 
+    const getSvgWithUniqueIds = (code: string) => {
+        // A workaround to ensure that important ids in svg <defs> are unique to avoid issues with rendering in Storybook Docs:
+        const unique_id = Math.floor(Math.random() * 1000);
```

Comment:
```suggestion
        const unique_id = Math.floor(Math.random() * 1000);
```
Can we use something like `uuid` https://www.npmjs.com/package/uuid. There's a small chance that the `unique` id will overlap even with `Math.random`

Reply:
@yashim-deriv thank you for the brilliant idea! I have installed & used it 🙂